BrainPortal/app/controllers/nh_users_controller.rb

```diff
@@ -67,7 +67,7 @@ def edit #:nodoc:
   def change_password #:nodoc:
     @user = current_user
     if user_must_link_to_oidc?(@user)
-       cb_error "Your account can only authenticate with Globus identities.", :redirect => { :action => :myaccount }
+       cb_error "Your account can only authenticate with OpenID identities (such as Globus).", :redirect => { :action => :myaccount }
```

Comment:
Remove the ' (such as Globus)' given a CBRAIN installation might not have a globus OIDC configured at all, and it could be confusing for the user.

Reply:
This is not 'resolved' ?!? Please don't just ignore my code reviews like that.

Reply:
sorry, I jumped the gun. I guess my push failed yesterday or smth